commands/settings/oidc/configure.go

```diff
@@ -33,7 +34,7 @@ var (
 		Args:    cobra.NoArgs,
 		Long:    "Configure OIDC settings",
 		Short:   "Configure OIDC settings",
-		Example: `everestctl settings oidc configure --issuer-url https://example.com --client-id 123456`,
+		Example: `everestctl settings oidc configure --issuer-url https://example.com --client-id 123456 --scope "openid profile email groups"`,
```

Comment:
I would make format the same as for --namespaces flag, i.e
```suggestion
		Example: `everestctl settings oidc configure --issuer-url https://example.com --client-id 123456 --scopes openid,profile,email,groups`,
```

And in addition this is less error-prone way to pass values